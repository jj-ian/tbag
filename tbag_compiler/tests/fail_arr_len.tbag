true {
	int a;
	int len;
	len = arr_len(a);
	print(len);
	endgame;
}
