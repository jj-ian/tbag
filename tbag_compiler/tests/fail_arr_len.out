Fatal error: exception Failure("arr_len expects an array
                      argument")
